### DISCUSSION TITLE
Impact Measurement

### DISCUSSION THREAD

**Discussion by ryscheng** [hypercerts-org/hypercerts#640] [Ideas - where HIPs are born]
Thank you Corey for assembling this awesome list of existing resources. Going to start a thread so that we can all keep adding interesting impact measurements that can help inform evaluations

Resources to pull usage or community metrics for OSS projects:

- [ecosyste.ms](http://ecosyste.ms/) - andrew nesbitt's legendary crawler libraries
- [deps.dev](http://deps.dev/) -  service repeatedly examines sites such as [github.com](http://github.com/), [npmjs.com](http://npmjs.com/), and [pkg.go.dev](http://pkg.go.dev/) to find up-to-date information about open source software packages
- [repology.org](http://repology.org/) - monitors package repos
- [nebula crawler](https://github.com/dennis-tra/nebula) ([reports](https://github.com/dennis-tra/nebula-crawler-reports)) - A libp2p [DHT](https://en.wikipedia.org/wiki/Distributed_hash_table) crawler and monitor that tracks the liveness of peers. The crawler connects to DHT bootstrap peers and then recursively follows all entries in their [k-buckets](https://en.wikipedia.org/wiki/Kademlia) until all peers have been visited.
- [GitHub API](https://docs.github.com/en/rest/overview/about-githubs-apis?apiVersion=2022-11-28) ([REST](https://docs.github.com/en/rest?apiVersion=2022-11-28) & [GraphQL](https://docs.github.com/en/graphql/overview/about-the-graphql-api)): [Community](https://docs.github.com/en/rest/metrics/community?apiVersion=2022-11-28) | [Stats](https://docs.github.com/en/rest/metrics/statistics?apiVersion=2022-11-28) | [Traffic](https://docs.github.com/en/rest/metrics/traffic?apiVersion=2022-11-28)
- [Gitcoin grant data](https://www.tokenomics.io/gitcoin)

PLN Specific Resources:

- [Sentinel](https://lilium.sh/)
  - Stewards of [lily](https://lilium.sh/software/lily/introduction/), a wrapped lotus node designed specifically for indexing the Filecoin blockchain
  - Working on queryable, lightweight historical snapshots of FVM chainstate
- [Probelab](https://research.protocol.ai/groups/probelab/) is PL's Protocol Benchmarking & Optimization Team
  - Links: [Notion Page of Projects](https://www.notion.so/pl-strflt/ProbeLab-Protocol-Benchmarking-Optimization-a63238fd1b184d6f8fea4bb38d975208) | [GitHub Repo](https://github.com/protocol/network-measurements) | [Network Measurement Reports](https://github.com/protocol/network-measurements/tree/master/results)
- FMV-Specific: 
  - [Filecoin APIs & Explorers](https://docs.filecoin.io/networks/mainnet/explorers/) Overview Docs page
  - Note: as of May 2023, Filfox is the most reliable explorer.
- IPFS-Specific:
  - [IPFS Gateways](https://docs.ipfs.tech/concepts/ipfs-gateway/) provide an HTTP-based service that allow HTTP-incompatible browsers, tools and software to access IPFS content.
  - [IPFS Ecosystem dashboard](https://ipfs.ecosystem-dashboard.com/)
  - [IPFS Ecosystem map](https://ecosystem.ipfs.tech/showcase/?category=focus)
  - [Probe Lab GitHub](https://github.com/plprobelab) | [Tiros IPFS Measurement tool](https://github.com/plprobelab/tiros)
  -

**Comment by marsrobertson** [comment]
> interesting impact measurements that can help inform evaluations

# Impact Genome Registry

The most exciting part migrated / requires login, here it is on Web Archive: http://web.archive.org/web/20230307121254/https://impactgenome.org/impact-index/

<img width="1117" alt="image" src="https://github.com/hypercerts-org/hypercerts/assets/44369284/fb58cc61-55a2-4116-a1f4-972e1d9e704f">

# IMF

![image](https://github.com/hypercerts-org/hypercerts/assets/44369284/d82ce636-8f4b-4363-8348-4db668b736b6)
https://www.imf.org/en/Publications/fandd/issues/2020/09/how-african-elephants-fight-climate-change-ralph-chami

![image](https://github.com/hypercerts-org/hypercerts/assets/44369284/c2a5b765-1a0a-4ead-b023-da5cdae5782d)
https://www.imf.org/en/Publications/fandd/issues/2019/12/natures-solution-to-climate-change-chami

**RELATED:** https://www.theguardian.com/environment/2022/jul/18/wild-bison-return-to-uk-for-first-time-in-thousands-of-years

_(reintroduction of Bison in the UK to provide ecosystem services)_

# Basin

Captured my attention on Twitter but they are just getting started

<img width="584" alt="image" src="https://github.com/hypercerts-org/hypercerts/assets/44369284/b6dc7685-1076-4182-8ae4-1fefcc083398">
https://twitter.com/tmoindustries/status/1655547105803485185


# [BeZero](https://bezerocarbon.com/listings/) - rating carbon credits

<img width="980" alt="image" src="https://github.com/hypercerts-org/hypercerts/assets/44369284/359a479d-086e-4d3e-a5ee-baee83bdb921">

I'm in the "don't trust verify"

# [Solid World](https://www.solid.world/crisp) - rating carbon credits

Attended Twitter Spaces recently, they are doing it as "public good"

# https://simbiotica.finance/#explore-tokens

<img width="1377" alt="image" src="https://github.com/hypercerts-org/hypercerts/assets/44369284/73b6e909-e231-426a-bfde-e110c6cc20f8">

# dMRV

* https://mrv.earth/
* Haven't tried but appeared on Gitcoin grants: https://explorer.gitcoin.co/#/round/1/0x421510312c40486965767be5ea603aa8a5707983/0x421510312c40486965767be5ea603aa8a5707983-78

# 🛰 https://www.gainforest.earth/

My personal research: https://basex.notion.site/Carbon-Credit-Markets-94abec41f8b34a2889cafd3fbc84c76c

_(didn't update in a few months)_

<img width="722" alt="image" src="https://github.com/hypercerts-org/hypercerts/assets/44369284/dc7fc762-bf86-4e6c-97c9-b792c4c17dce">

# BaseX.com  - new definition of value

Personal shill.

That's why I'm so passionate about the problem space.

<img width="827" alt="image" src="https://github.com/hypercerts-org/hypercerts/assets/44369284/f8c5402c-b5d3-4222-bc50-8a6f77125023">

New definition of value and measuring impact = pretty much the same.

Check some of the comments / suggestions / ideas here: https://github.com/hypercerts-org/hypercerts/discussions/449

Two slides from my pitch deck that illustrate the problem space:

<img width="1015" alt="image" src="https://github.com/hypercerts-org/hypercerts/assets/44369284/81737d2f-efd9-40c0-9c26-064470d57aba">

<img width="1044" alt="image" src="https://github.com/hypercerts-org/hypercerts/assets/44369284/f21962e8-8809-451e-91d8-6d22c04ae983">

**Reply by ccerv1** [reply]
This is fantastic, thank you for the added resources @marsrobertson !

**Comment by marsrobertson** [comment]
Thread: https://twitter.com/LBorreani/status/1658738167434948608

Blog: https://medium.com/kolektivo-co/monitoring-reporting-and-verification-methodologies-in-regenerative-finance-81d76cb9710

Video: https://www.youtube.com/watch?v=joayKDnNU6g

Paper: https://docsend.com/view/avd6hzqqhfs2hp2p

<img width="514" alt="image" src="https://github.com/hypercerts-org/hypercerts/assets/44369284/f0a315c2-48c6-4a7c-8d28-14ebb313e03e">

<img width="479" alt="image" src="https://github.com/hypercerts-org/hypercerts/assets/44369284/81279389-a8d5-4ebd-a22e-984ce32424e4">

Here another bunch of links, found referenced in the report: https://mrvcollective.org/

Initially posted to Regens Unite: https://t.me/regensunite/2653
<img width="687" alt="image" src="https://github.com/hypercerts-org/hypercerts/assets/44369284/3f3f3e21-a731-4c68-bc49-a310f3a2413f">

**EDIT / UPDATE:**

I had a call with a person who did the report, then joined yet another call:

<img width="879" alt="image" src="https://github.com/hypercerts-org/hypercerts/assets/44369284/a3d8b6c5-3e60-4e5e-85c0-36ef2a118923">

As of May 2023 these guys seem to be leading the pack in terms of dMRV
- https://twitter.com/dMeter4dmrv
- https://discord.gg/wvA99DHgUB
- Miro board with sprint planning: ........ _(actually not sharing to prevent vandalism, join the discord, you'll find it)_

**Comment by ryscheng** [comment]
Couple more for open source software

https://github.com/dapplooker

https://github.com/pingcap/ossinsight-lite
